src/dynamicprompts/parser/parse.py

```diff
@@ -227,13 +237,22 @@ def create_parser() -> pp.ParserElement:
     return prompt
 
 
-def parse(prompt: str) -> Command:
+def parse(prompt: str, braces="{}") -> Command:
```

Comment:
There's technically nothing preventing the use of any arbitrary strings as separators (e.g. `#| |#` could work)..?
```suggestion
def parse(
  prompt: str,
  *,
  braces: Collection[str]=("{", "}"),
) -> Command:
```

Also probably worth it to make that kwarg-only for future expansion.